assignments/life_expectancy/cleaning.py

```diff
@@ -52,6 +53,8 @@ def clean_data(df: DataFrame, region: str) -> DataFrame:
     df_clean = df_unpivot.dropna()
 
     # filter dataframe by desire region
-    df_clean_filt = df_clean[df_clean['region'] == region]
+    df_clean_filt = df_clean[df_clean['region'] == region.value]
 
```

Comment:
Minor suggestion (conditioned on interpretation of the assignment): I considered matching the region in both upper and lower case (e.g., if the input df contains one row with region "PT" and another with "pt", both would be selected). Not sure if this is the goal of the assignment or not; anyway it is a minor point